packages/application-generic/src/usecases/select-integration/select-integration.usecase.ts

```diff
@@ -66,6 +66,14 @@ export class SelectIntegration {
       active: true,
     };
 
+    if (command.identifier) {
+      query = {
+        _organizationId: command.organizationId,
+        channel: command.channelType,
+        identifier: command.identifier,
+      };
+    }
+
```

Comment:
For now, allow also disabled integrations. Until we allow more than one active instance per environment.

Reply:
I see now that we do allow 2 active for same env. I will check again

Reply:
Changed it to allow to override only with active instances